lib/Config/CLIConfig.cpp

```diff
@@ -13,39 +13,48 @@
 //===----------------------------------------------------------------------===//
 
 #include "Config/CLIConfig.h"
+#include "Config/QSSConfig.h"
+
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Error.h"
+
+#include <string>
 
 using namespace qssc::config;
 
+namespace {
 // The space below at the front of the string causes this category to be printed
 // first
-static llvm::cl::OptionCategory
+llvm::cl::OptionCategory
     qsscCat_(" QSS Compiler Options",
              "Options that control high-level behavior of QSS Compiler");
 
-llvm::cl::OptionCategory &qssc::config::getQSSCCategory() { return qsscCat_; }
-
-static llvm::cl::opt<std::string> configurationPath(
+llvm::cl::opt<std::string> configurationPath(
     "config",
     llvm::cl::desc("Path to configuration file or directory (depends on the "
                    "target), - means use the config service"),
     llvm::cl::value_desc("path"), llvm::cl::cat(qsscCat_), llvm::cl::Optional);
 
-static llvm::cl::opt<std::string>
+llvm::cl::opt<std::string>
     targetStr("target",
               llvm::cl::desc(
                   "Target architecture. Required for machine code generation."),
               llvm::cl::value_desc("targetName"),
               llvm::cl::cat(getQSSCCategory()));
 
-static llvm::cl::opt<bool> allowUnregisteredDialects(
+llvm::cl::opt<bool> allowUnregisteredDialects(
```

Comment:
Same issue with coding standards here and static vs anonymous namespaces.

Reply:
Same as above.